app/config.yml

```diff
@@ -1,7 +1,8 @@
 
 default:
-  stackName: test-cumulus-integration
-  stackNameNoDash: TestCumulusIntegration
+  prefix: change-me 
+  stackName: '{{prefix}}-integration'
+  # stackNameNoDash: TestCumulusIntegration
```

Comment:
Should we remove this if it's not needed?

Reply:
I commented it out so it fails if someone tries to deploy the default. but I think it's useful to have it there so people know they have to set it.